DarkData_Python/Rules_from_experts/Measurement_Rules.py

```diff
@@ -44,7 +61,7 @@ def fun(i,x):
         "(?event rdf:type dd:"+events[i]+"),"+"\n"+
         "(?candidate dd:candidateVariable ?variable),"+"\n"+
         "(?variable dd:spatialResolution ddspatial:"+str(Rules['Measurement'][measurements[j]].name).replace(' ','+')+"),"+"\n"+
```

Comment:
As in the example rules, this should be ``dd:measurement`` instead of ``dd:spatialResolution``